README.md

```diff
@@ -66,7 +80,173 @@ Sprint Sizing Legend:
+│   ├── logging.conf                  <- Configuration files for python loggers
+│   ├── config.py                     <- Contains all other configurations
+│
+├── data                              <- Folder that contains data used or generated. Only the external/ and sample/ subdirectories are tracked by git. 
+│   ├── raw/                          <- Place to put raw data used for training the model 
+│   ├── usage_log/                    <- Contains sqlite database usage_log.db which tracks all usage statistics
+│
+├── deliverables                      <- Contains all deliverables.
+│
+├── docs                              <- A default Sphinx project; see sphinx-doc.org for details.
+│
+├── figures                           <- Generated graphics and figures to be used in reporting.
+│
+├── models                            <- Trained model objects (TMOs), model predictions, and/or model summaries
+│   ├── archive                       <- No longer current models. This directory is included in the .gitignore and is not tracked by git
+│
+├── notebooks
+│   ├── develop                       <- Current notebooks being used in development.
+│   ├── deliver                       <- Notebooks shared with others. 
+│   ├── archive                       <- Develop notebooks no longer being used. 
+│
+├── src                               <- Source data for the project 
+│   ├── archive/                      <- No longer current scripts.
+│   ├── helpers/                      <- Helper scripts used in main src files 
+│   ├── sql/                          <- SQL source code
+│   ├── load_data.py                  <- Script for downloading data from the input source 
+│   ├── log_usage_data.py             <- Script for building the usage log database and injesting data in it
+│
+├── test                              <- Files necessary for running model tests 
+│
+├── run.py                            <- Simplifies the execution of one or more of the src scripts 
+│
+├── requirements.txt                  <- Python package dependencies 
+```
+
+## Running the application
+
+Ths application can be run on both local system as well as on AWS. Steps on how to deploy the app for both settings is given below.
+
+### 1. Set up environment 
+
+The `requirements.txt` file contains the packages required to run the model code. An environment can be set up in two ways. 
+
+#### With `virtualenv`
+
+```bash
+pip install virtualenv
+
+virtualenv housePrices
+
+source housePrices/bin/activate
+
+pip install -r requirements.txt
+
+```
+#### With `conda`
+
+```bash
+conda create -n housePrices python=3.7.3
+conda activate housePrices
+pip install -r requirements.txt
+
+```
+
+### 2. Download the data
+
+Original Data Source: [Kaggle](https://www.kaggle.com/c/house-prices-advanced-regression-techniques)
+
+For the ease of downloading, the raw data has been downloaded and placed in a public s3 bucket: **s3://housing-prices-data** 
```

Comment:
It will be great if you could instead put this snippet below the instruction as well as let the person running your code know that they need to have AWS setup and connected via root key to run from local.